src/main/drivers/serial_uart_pinconfig.c

```diff
@@ -46,6 +46,12 @@ void uartPinConfigure(const serialPinConfig_t *pSerialPinConfig)
     for (size_t hindex = 0; hindex < UARTDEV_COUNT_MAX; hindex++) {
```

Comment:
Sorry, I overlooked this ... 
I would still prefer defining all USE_USARTx that can be user-configured, eventually removing it. It should be safer with rest of code:
- serial.c : serialPortIdentifier_e won't contain `SERIAL_PORT_USARTx`
- interrupt handlers
- initInverters
- serial_pinconfig default (this one is probably OK, it may be split from USE_USARTx)